amostra/tests/test_revert.py

```diff
@@ -0,0 +1,33 @@
+import amostra.mongo_client
+from hypothesis import given, strategies as st
+from hypothesis.strategies import text
+from hypothesis import settings
+from pymongo import MongoClient
+import random
+import uuid
+
+
+alphabet_list = ''
+for i in range(26):
+    alphabet_list = alphabet_list + chr(97 + i)
+
+
+@given(names = st.lists(st.text(alphabet=alphabet_list, min_size=1, max_size=4), min_size=3, max_size=4, unique=True))
+@settings(max_examples = 10, deadline = 1000)
+def test_revert(client, names):
+    n = len(names)
+    s = client.samples.new(name = names[0])
+    uuid = s.uuid
+    for name in names[1:]:
+        s.name = name 
+
+    num = random.randint(0, n-2)
+
+    revert_target_cursor = client._db.samples_revisions.find({'revision': num, 'uuid': uuid})
+    s.revert(num)
+    target = next(revert_target_cursor)
+    for name, trait in s.traits().items():
+        if name is 'revision':
```

Comment:
Use `==` for string comparison. It's a fluke that `is` happens to work here. It won't always.